syntax.php

```diff
@@ -83,6 +83,9 @@ function handle($match, $state, $pos, Doku_Handler $handler){
         $urlparam = array();
         foreach($paramm as $key => $value) {
             switch($key) {
+                case 'list':
+                    $urlparam[] = $key . '=' . $paramm[$key];
```

Comment:
since the set parameter is later printed without any further sanitation, it needs to be sanitized here. The way it is implemented currently would open an XSS vulnerability